src/factors/PartialPose3.jl

```diff
@@ -113,7 +113,29 @@ getManifold(::Pose3Pose3XYYaw) = SpecialEuclidean(2)
 ## NOTE, Yaw only works if you assume a preordained global reference point, such as identity_element(Pose3)
 function (cfo::CalcFactor{<:Pose3Pose3XYYaw})(X, wTp, wTq )
   #
+  #FIXME if else is for comparing 2 different aprouches in factor
+  if true
+    
+  M = SpecialEuclidean(2)
 
+  rx = normalize(wTp.parts[2][1:2, 1])
```

Comment:
https://github.com/JuliaRobotics/RoME.jl/commit/85a997fc37973b71a5d20f3b729e40f984bb98bf#r57341417

Reply:
Hi Johan, did you see my comment on why i think there is a problem with sub-selecting from rotation matrix?  This does not give just yaw, pitch is also mixed in on the first column.